README.md

```diff
@@ -53,7 +53,9 @@ wt create https://raw.githubusercontent.com/auth0/wt-cli/master/sample-webtasks/
 ```bash
 wt create https://raw.githubusercontent.com/auth0/wt-cli/master/sample-webtasks/express.js \
           --name express \
-          --no-parse --no-merge
+          --no-parse \
```

Comment:
@rdaniels6813 thanks so much for the submission!

While we're at it, `--no-parse` has also been [deprecated](https://github.com/auth0/wt-cli/blob/ca2374c2c119070aaabb818c06a24ff4b5c28be0/bin/create.js#L102). could you please just remove this line? Thanks!

Reply:
Removed!